gr-blocks/include/gnuradio/blocks/file_source.h

```diff
@@ -1,6 +1,6 @@
 /* -*- c++ -*- */
 /*
- * Copyright 2012 Free Software Foundation, Inc.
+ * Copyright 2018 Free Software Foundation, Inc.
```

Comment:
The important part here really is the *first* copyright date, and to a much lesser extent the last; so, never replace OLDYEAR with NEWYEAR, but just append (as @mbr0wn said)